README.md

```diff
@@ -30,8 +30,9 @@ sfncli -activityname sleep-100 -region us-west-2 -workername sleep-worker -cmd s
 - On startup, call [`CreateActivity`](http://docs.aws.amazon.com/step-functions/latest/apireference/API_CreateActivity.html) to register an [Activity](http://docs.aws.amazon.com/step-functions/latest/dg/concepts-activities.html) with Step Functions.
 - Begin polling [`GetActivityTask`](http://docs.aws.amazon.com/step-functions/latest/apireference/API_GetActivityTask.html) for tasks.
 - Get a task. Take the JSON input for the task and
-  - if it's an array, use this as the args to the command.
-  - if it's anything else (e.g. JSON object), it's currently not supported
+  - if it's a JSON object, use this as the last arg to the command.
+  - if it's anything else (e.g. JSON array), an error is thown
+  - if JSON object has a `_EXECUTION_NAME` property, an corresponding env var called `_EXECUTION_NAME` is add to the sub-process enviornment
```

Comment:
nits:
`has an`
`a corresponding`
`is added`
`environment`

Reply:
Fixed in 7f321d3a45b9ad7376eea3fde6c47ce992bfcf0f